demo/sync/syncServices.js

```diff
@@ -1,8 +1,17 @@
 var config = require('./config.json');
 var _ = require('lodash');
-angular.module('wfm.sync',[]).service('syncService', ['$http', '$window', 'syncPool', 'userService', function($http, $window, syncPool, userService) {
-  return userService.getProfile($http, $window)
-    .then(syncPool.syncManagerMap);
+angular.module('wfm.sync',[]).service('syncService', ['$http', '$window', 'syncPool', 'Auth', function($http, $window, syncPool, Auth) {
+  if (Auth.keycloak) {
+    // retrieve the users profile from keycloak
+    return Auth.keycloak.loadUserProfile()
+      .success(syncPool.syncManagerMap)
```

Comment:
When verifying keycloak still works, it hits this error after login:
```
angular.js:14525 TypeError: syncService.then is not a function
    at new SyncApiDataService (syncDataRepository.js:5)
    at new <anonymous> (syncServices.js:23)
```
Not 100% sure on the cause but the `syncPool.syncManagerMap` is using the following for filtering:

```
var filter = {
    'assignee': profileData.id
  };
```
For Keycloak users, the raincatcher user ID is under `profile.attributes.name[0]`, so the `profile.id` will only return a keycloak user ID, which isn't what we want i don't think?